src/plugins/walletConnectToDapps/v1/components/modals/connect/ConnectContent.tsx

```diff
@@ -69,7 +69,10 @@ export const ConnectContent: React.FC<ConnectContentProps> = ({ handleConnect })
           <Heading flex={1} fontSize='xl'>
             <Text translation='plugins.walletConnectToDapps.modal.connect.title' />
           </Heading>
-          <Link href='#' target='_blank'>
+          <Link
+            href='src/plugins/walletConnectToDapps/v1/components/modals/connect#'
```

Comment:
?

Reply:
Not ready for review mate! 😅

But I'd say I missed a CMD + Shift + O when attempting to open a file.